libraries/spacy-3.0.x/Dockerfile

```diff
@@ -0,0 +1,9 @@
+
+RUN pip install "spacy>=3.0,<3.1"
+RUN python -m spacy download en_core_web_sm
+RUN python -m spacy download de_core_news_sm
+RUN python -m spacy download es_core_news_sm
+RUN python -m spacy download pt_core_news_sm
+RUN python -m spacy download fr_core_news_sm
+RUN python -m spacy download it_core_news_sm
+RUN python -m spacy download nl_core_news_sm
```

Comment:
How large are these languages?

Reply:
> How large are these languages?

Looks like the largest is 22 MB, and the smallest is 12 MB; overall very lightweight, which is great! Unfortunately not super easy to install programmatically from within a spacy script as they really like their CLI :joy: